### ISSUE TITLE
ProjectApi removeMember() throws GitLabApiException at success case

### ISSUE THREAD

**Issue by spingaroliver** [gmessner/gitlab4j-api#29]
When I remove a member from a project with projectApi.removeMember(); function the user is removed, but I get GitLabApiException: No Content exception.
The GitLab API returns 204 - No Content when the deletion is success.

So change this:
```java
public void removeMember(Integer projectId, Integer userId) throws GitLabApiException {
        delete(Response.Status.OK, null, "projects", projectId, "members", userId);
    }
```
to this

```java
public void removeMember(Integer projectId, Integer userId) throws GitLabApiException {
        delete(Response.Status.NO_CONTENT, null, "projects", projectId, "members", userId);
    }
```

I think other delete() calls is bad too.

**Comment by spingaroliver** [comment]
Same problem at GroupApi removeMember() @

**Comment by gmessner** [comment]
Changed the expected HTTP status to NO_CONTENT (dependent on GitLab API version).  Have also made same change in all sub API's.  Fix is included in the new 4.2.3 release.
